### DISCUSSION TITLE
running platform android in capacitor issue

### DISCUSSION THREAD

**Discussion by ourwa792** [ionic-team/capacitor#7447] [Q&A]
am new in the web,android development world. I am getting started on a project in Capacitor js, followed the instructions in the documentation when I run cap run android it crushed Running Gradle build - failed! [error] Starting a Gradle Daemon, 2 incompatible and 1 stopped Daemons could not be reused, use --status for details

    FAILURE: Build failed with an exception.

    * What went wrong:
    A problem occurred configuring root project 'android'.
    > Could not resolve all files for configuration ':classpath'.
    > Could not resolve com.android.tools.build:gradle:8.2.1.
    Required by:
    project :
    > No matching variant of com.android.tools.build:gradle:8.2.1 was found.
    The consumer was configured to find a library for use during runtime,
    compatible with Java 8, packaged as a jar, and its dependencies declared
    externally, as well as attribute 'org.gradle.plugin.api-version' with
    value '8.2.1' but:
    - Variant 'apiElements' capability com.android.tools.build:gradle:8.2.1
    declares a library, packaged as a jar, and its dependencies declared
It work on Android Studio.

windows 11
in my environment variables i have:  
ANDROID HOME :  C:\Users\user\AppData\Local\Android\Sdk
path: C:\Program Files\Java\jdk-20
      C:\Program Files\Java\jdk-20\bin
      C:\Program Files\Java\jre-1.8
JAVA HOME C:\Program Files\Java\jre-1.8

**Comment by jcesarmobile** [comment]
You have to use java 17 but you are using java 8

**Reply by ourwa792** [reply]
thanks, i have solved it 
have another questions
i want to use ionic  CLI and capacitor with vanilla js  without any framework
i cant find that in docx

**Reply by UcheWebDev** [reply]
how did you solve it?

**Reply by preetamjakhmola** [reply]
how did you solve it man,

**Reply by allnightru** [reply]
1. [Install Java 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html)
2. Change JAVA_HOME environent var to where you've installed it ("C:\Program Files\Java\jdk-17" for me).

**Comment by ourwa792** [comment]
u can upgrade java version or change java path to android studio path
